assets/js/app.js

```diff
@@ -15,3 +15,7 @@ import './vendor/es6.polyfill';
 // Our code
 import './ujs';
 import './when-ready';
+
+import '../css/themes/default.scss';
```

Comment:
Maybe I'm just being pedantic, but this is _so_ dumb. Is there a way to do this without literally importing the CSS files into the scripts?

Reply:
I tried adding them as separate entrypoints in the config caused files to be written to the output that only had webpack's runtime in it. You somehow have to tell webpack about these files' existence and this is the only way I know to do that.

Reply:
I updated this to use `resolve.alias` but then `~` is required in front of the alias names, that's the best that can be done by the looks of it.